### PR TITLE
Backend. Module Blog. Move summary field to top

### PR DESCRIPTION
**Pull request by Sellato**
- Enhancement

## Pull request description
I think the right order of fields is:
1. Summary
2. Image
3. MainContent

This PR move "Summary" field to the first place.

### CODE REVIEW

File: src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig [hunk 1]
```diff
@@ -146,7 +158,7 @@
                       <div class="form-group last">
                         <label for="tags" class="control-label">{{ 'lbl.Tags'|trans|ucfirst }}</label>
                         {% form_field tags %} {% form_field_error tags %}
-                        <div class="help-block">{{ 'msg.HelpTextTags'|trans }}</div>
+                        <div class="help-block" id="tags-info">{{ 'msg.HelpTextTags'|trans }}</div>
```
Comment (carakas) [inline src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig]: why did you add the id?

File: src/Backend/Modules/Blog/Layout/Templates/Add.html.twig [hunk 2]
```diff
@@ -121,7 +127,7 @@
                       <div class="form-group last">
                         <label for="tags" class="control-label">{{ 'lbl.Tags'|trans|ucfirst }}</label>
                         {% form_field tags %} {% form_field_error tags %}
-                        <div class="help-block">{{ 'msg.HelpTextTags'|trans }}</div>
+                        <div class="help-block" id="tags-info">{{ 'msg.HelpTextTags'|trans }}</div>
```
Comment (carakas) [inline src/Backend/Modules/Blog/Layout/Templates/Add.html.twig]: Why did you add the id?

File: src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig [hunk 3]
```diff
@@ -56,52 +56,64 @@
...
                   <div class="panel panel-default">
                     <div class="panel-heading">
-                      <h2>{{ 'lbl.Image'|trans|ucfirst }}</h2>
+                      <h2><label for="image">{{ 'lbl.Image'|trans|ucfirst }}</label></h2>
                     </div>
                     <div class="panel-body">
-                      {% if item.image %}
-                        <div>
-                          <img src="{{ FRONTEND_FILES_URL }}/Blog/images/128x128/{{ item.image }}" class="img-thumbnail" width="128" height="128" alt="{{ 'lbl.Image'|trans|ucfirst }}" />
+                      <div class="row">
```
Comment (carakas) [inline src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig]: nice improvement, thanks

**Comment by carakas** [comment]
@Sellato why the revert?

File: src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig (hunk 1 above)
Comment (Sellato) [reply src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig]: give me some time, something goes wrong :(

**Comment by Sellato** [comment]
something crashed in my repository...
please ignore all changes.
I try to got clean repository, sorry for that. I have not enough skills in github.
I'm very sorry.

**Comment by carakas** [comment]
@Sellato it looked good what you initially committed
